test/functional/rpc_verifyislock.py

```
@@ -3,7 +3,7 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-from test_framework.messages import CTransaction, from_hex, hash256, ser_compact_size, ser_string
+from test_framework.messages import CTransaction, tx_from_hex, hash256, ser_compact_size, ser_string
```

Flake8 (7.3.0)

[error] 'test_framework.messages.CTransaction' imported but unused

(F401)